CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ We use _breaking :warning:_ to mark changes that are not backward compatible (re
 
 ### Added
 
+- [#3740](https://github.com/thanos-io/thanos/pull/3740) Query: Added `--query.default-set` flag to set default step.
```

Comment:
@hitanshu-mehta A small typo. The flag is `query.default-step` instead of `query.default-set`. After modifying this then we are good to go!

Reply:
Opps. Thank you for pointing out 😄